package.json

```diff
@@ -8,7 +8,7 @@
     "test": "jest",
     "build": "gatsby build",
     "check-not-master": "[ \"$(git rev-parse --abbrev-ref HEAD)\" = \"master\" ] && echo \"\\033[31mDon't commit to master\" && exit 1 || exit 0",
-    "prod:build": "docker image build -t new-docs:localbuild .",
+    "prod:build": "cross-env TAG=new-docs:localbuild AMPLITUDE_KEY=$AMPLITUDE_KEY make docker-build",
```

Comment:
This brings `yarn production` very closely in line with how Jenkins runs a build. Lots of tools (including make, docker, and gatsby) clear out environment variables for security reasons, so this removes another mismatch and thus a potential source of production-only bugs.

It passes through an `AMPLITUDE_KEY` env var for ease of testing. Otherwise, it'd have to be a package.json modification which invalidates docker cache for installing dependencies—not ideal.